chapters/functions.tex

```diff
@@ -1880,10 +1880,11 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 "GenerateEvents" "=" ( false | true )
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-Has only an effect within a function declaration
-
-If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events (one possibility of doing this is to inline the function and generate events for the inlined function).
+Has only an effect within a function declaration.
 
+If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events.
```

Comment:
What @eshmoylova, @casella and I propose would be to remove the tool's freedom to respect `GenerateEvents` or not, something like this:
```suggestion
By default, \lstinline!GenerateEvents = false! and expressions in the function body that would normally be event-generating shall not generate events, similar to inlining the function body while wrapping all expressions in \lstinline!noEvent!, see \cref{modelica:noEvent}.
By specifying \lstinline!GenerateEvents = true!, event-generating expressions in the function body shall generate events as normal, similar to inlining the function body without wrapping all expressions in \lstinline!noEvent!.
```